spec/common/globals/environment.js

```diff
@@ -3,7 +3,7 @@
 define(function(require) {
 	var defaultLogLevel = 2,
 		environment = isBrowser ? (window.__env__ || {}) : process.env,
-		ablyEnvironment = environment.ABLY_ENV || 'sandbox',
+		ablyEnvironment = environment.ABLY_ENV || 'staging',
```

Comment:
Nope, was just to change the ci run. (Hence the commit message of `TEMP NOTFORMERGING` :) )